src/commands/aksCompareCluster/aksCompareCluster.ts

```diff
@@ -42,25 +44,21 @@ export default async function aksCompareCluster(_context: IActionContext, target
         return;
     }
     const containerServiceClient = getContainerClient(subscriptionNode.result);
-    const clusterList: string[] = [];
-    const clusterResourceDictionary: Dictionary<string> = {};
```

Comment:
@Tatsinnit you can just a leave a comment to mention why this file was changed, at first glance I thought this file had the bug then realized it was included as part of this PR to make the code simple and readable. Thanks.

Reply:
This is relating to kind testing from @sprab who actually find a small thing which needs adjustment.

Hence it will make this PR more simple and readable especially the `az compare` feature :) we have simplified the whole logic, but if you are wanting to understand the feature, let me know I can help to do that as well.

Hope this helps, thanks heaps for comment. fyi:  @hsubramanianaks, please let me know if you have any further query or want me to detail the eventual Simplification in detail for az compare feature.

Reply:
@Tatsinnit Thank you for the detailed explanation, looks clear to me.